bin/run-command.sh

```diff
@@ -18,10 +18,101 @@ set -euo pipefail
 # TODO: Add ability to change task IAM Role. Part 3 of multipart update https://github.com/navapbc/template-infra/issues/354#issuecomment-1693973424
 # TODO: Change to keyword arguments. Part 3 of multipart update https://github.com/navapbc/template-infra/issues/354#issuecomment-1693973424
 
-APP_NAME="$1"
-ENVIRONMENT="$2"
-COMMAND="$3"
-ENVIRONMENT_VARIABLES=${4:-""}
+ACCOUNT_ID="$(./bin/current-account-id.sh)"
+
+while :; do
+  case $1 in
+    -a|--app|--app-name)
```

Comment:
let's keep it simple and just provide one flag option with it full spelled out

Reply:
Can do but this is still in progress. Don't feel pressured to monitor and fix this